cdap-ui/app/cdap/components/ConfigurationGroup/utilities/index.ts

```diff
@@ -22,6 +22,7 @@ import {
 } from 'components/ConfigurationGroup/types';
 import xor from 'lodash/xor';
 import flatten from 'lodash/flatten';
+import { string } from 'prop-types';
```

Comment:
This is not required, `string` is primitive type

Reply:
Removed.